datajoint/jobs.py

```diff
@@ -87,6 +87,32 @@ def reserve(self, table_name, key):
             return False
         return True
 
+    def ignore(self, table_name, key):
+        """
+        Set a job to be ignored for computation.  When a job is ignored, the job table contains an entry for the
+        job key, identified by its hash, with status "ignore".
+
+        :param table_name: `database`.`table_name`
+        :param key: the dict of the job's primary key
+        :return: True if ignore job successfully. False = the jobs is already taken
```

Comment:
Would you update this to a [Google Style](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) docstring? That is what we are supposed to be using for API docs w/ MkDocs Material.